client/web/compose/src/components/PageBlocks/RecordListBase.vue

```diff
@@ -161,9 +161,9 @@
                       <span
                         class="text-primary font-weight-bold"
                       >
-                        {{ !f.value.start ? 'NULL' : '' }}
-                        AND
-                        {{ !f.value.end ? 'NULL' : '' }}
+                        {{ !f.value.start ? $t('recordList.filter.conditions.null') : '' }}
+                        {{ $t('recordList.filter.conditions.and') }}
```

Comment:
the `AND` should be lowercase if its inside the filter to make it more text like.